page-about.php

```diff
@@ -12,6 +12,21 @@
     <?php 
       if ( have_posts() ) :
         while ( have_posts() ) : the_post();
+
+        // check if we have setupshop db fields
+        if (!get_post_meta( the_ID(), 'setupshop_interventions_focus', true)) {
+          add_post_meta( the_ID() , 'setupshop_interventions_focus' , 'education' , false ); 
+          add_post_meta( the_ID() , 'setupshop_interventions_focus' , 'health' , false );
+          add_post_meta( the_ID() , 'setupshop_interventions_focus' , 'games' , false );
+        }
```

Comment:
So, while busy, I was working in functions.php, Matt suggested, since it'll only be needed in the about page, let's put it there.
Do you mean, I completely went the wrong way about it?